views/content/wallet.md

```diff
@@ -0,0 +1,14 @@
+## Ethereum Wallet
+
+The Ethereum Wallet app that you can download allows you to run the full block chain on your own computer. It allows you very easy to be part of the blockchain.
+
+![](/images/wallet-screenshot.png)
+
+With the wallet app you can create new wallets and transfer ether really easily. You can also interact with smart contracts that other have deployed to the Ethereum blockchain.
+
+You don't absolute need the wallet to hold ether. There is different places that also allow you to interact with the Ethereum blockchain such as projects like
```

Comment:
I feel that pointing to other projects should be elsewhere in the home page, let's remove it from here.